src/features/amUI/requestPage/RequestsPage.tsx

```
@@ -1,18 +1,140 @@
+            name={consent.toParty.name}
+            type={'company'}
+            linkIcon
+            description={`${actionText} (${formatDateToNorwegian(consent.createdDate)})`}
+            as={(props) => (
+              <Link
+                to={`/consent/request?id=${consent.id}`}
+                {...props}
+              />
+            )}
+            controls={t('request_page.process_request')}
+          />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)